src/config/configuration.js

```diff
@@ -61,6 +62,7 @@ class Config {
       this.METRICS = process.env.METRICS !== "false"; // default: true
       this.METRICS_PORT = process.env.METRICS_PORT || 3000;
       this.FASTSYNC = process.env.FASTSYNC !== "false";  // default: true
+      this.IPFS_GATEWAY = process.env.IPFS_GATEWAY || "https://ipfs.io/ipfs/";
```

Comment:
I don't mind config the IPFS_GATEWAY as param, but i think this should continue in the manifest file as the default value and not in the code itself.